client/src/pages/Signup.js

```diff
@@ -0,0 +1,149 @@
+import React, {Fragment, useState} from "react";
+import {TextField, Button, OutlinedInput, FormHelperText, Typography, Box, Grid} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import {Link, useHistory} from "react-router-dom";
+import { useForm } from 'react-hook-form';
+// import {useAuth} from "../providers/auth/auth.provider";
+import axios from 'axios';
+// import {fetchUserFailure, fetchUserRequest, fetchUserSuccess, setIsAuthenticated} from '../providers/auth/auth.action';
+
+
+const AuthForm = (props) => {
```

Comment:
I think this component should be on a file of its own

Reply:
I did move these functions into a file on /src/components/auth.js